src/general/onboarding-merchants.md

```diff
@@ -23,7 +23,8 @@
 There are a number of merchant solutions to help vendors take payment in bitcoin and optionally convert to local fiat:
 
 - [Coinos](https://coinos.io/)
-- [Wallet of Satoshi](https://www.walletofsatoshi.com/)
+- [Wallet of Satoshi POS](https://www.walletofsatoshi.com/pos)
+- [Breez POS](https://breez.technology/mobile/) (non-custodial)
```

Comment:
```suggestion
- [Breez](https://breez.technology/mobile/)
```
Let's either remove `(non-custodial)` or add that tag to any of the other options that are also non-custodial.

Reply:
I think it would be good to mention it, but I does not know most of the others.